src/EnergyPlus/UnitarySystem.cc

```diff
@@ -12452,17 +12450,21 @@ namespace UnitarySystems {
                             if ((state.dataLoopNodes->Node(OutletNode).Temp - DesOutTemp) < Acc) break;
                         }
 
-                        Par[1] = double(this->m_CoolingCoilIndex);
-                        Par[2] = DesOutTemp;
-                        // dehumidification mode = 0 for normal mode, 1+ for enhanced mode
-                        // need to test what happens when Alt mode doesn't exist, or somehow test for it,
-                        // or fatal out in GetInput
-                        Par[3] = 1.0; // DehumidMode
-                        Par[4] = double(FanOpMode);
-                        Par[5] = this->m_CoolingSpeedNum;
-                        Par[6] = 1.0; //  this->m_CoolingSpeedRatio;
-                        Par[7] = 0.0;
-                        General::SolveRoot(state, Acc, MaxIte, SolFla, PartLoadFrac, &this->genericDXCoilResidual, 0.0, 1.0, Par);
+                        std::array<Real64, 8> Par2 = {
+                            double(this->m_CoolingCoilIndex),
+                            DesOutTemp,
```

Comment:
Is this assignment format done for performance reasons? It makes it a little harder to read if one wants to compare the `Par` elements in the residual function back to the values/variables passed in here.

Reply:
As I said above, I'm fine using individual assignments.  When the assignment is a clear set of values, the initializer list is, practically speaking, just a nicer look.